plugins/field-bitmap/src/field-bitmap.ts

```diff
@@ -473,29 +470,63 @@ export class FieldBitmap extends Blockly.Field<number[][]> {
   }
 
   /**
-   * Called when a mousedown event occurs within the bounds of a pixel.
+   * Checks if a down event is on a pixel in this editor and if it is starts an
+   * edit gesture.
+   *
+   * @param e The down event.
+   */
+  private onPointerStart(e: PointerEvent) {
+    const currentElement = document.elementFromPoint(e.clientX, e.clientY);
```

Comment:
Does `e.target` not work here?

Reply:
And in `onPointerMove`

Reply:
Touch events get captured by the element they start on. This means the target is always the element the event started on, so it won't change when you drag across another pixel cell.